src/main/java/org/betterx/betterend/entity/SilkMothEntity.java

```diff
@@ -46,7 +51,7 @@
 import java.util.EnumSet;
 import org.jetbrains.annotations.Nullable;
 
-public class SilkMothEntity extends DespawnableAnimal implements FlyingAnimal {
```

Comment:
Presumably if you're making a silk moth farm, having to nametag all your moths to prevent them from despawning would be a pain in the toucans.